rfcs/text/0001-wildcard-data-type.md

```diff
@@ -432,28 +372,28 @@ Stage 2: Document new concerns or resolutions to previously listed concerns. It'
+Performance and storage characteristics between wildcard and keyword will be different[4]. This difference will vary based deployment size and/or the amount of field data duplication. Fields that were previously indexed as keyword will switch to wildcard. With these fields now indexed as wildcard, users will be querying fields that are indexed as keyword in some indices and as wildcard in others. Any possible indexing or querying differences need to be understood and captured.
 
-The performance characteristics for both indexing and querying for the two types were explored and observations [noted](#comparison-with-keyword) in the earlier phases of this proposal. However, after additional benchmarking the increases in storage costs and decreasing in index performance was found to be significant enough that we need to revisit our approach. Each candidate field will need to be reassessed against a more rigorous set of criteria for inclusion into this initial type migration.
+Indexing and query performance characteristics of both types were explored. The observations were [noted](#comparison-with-keyword) earlier in this proposal. However, after additional benchmarking, the increases in storage costs and decreasing in index performance were found to be significant enough that we need to revisit our approach.
 
-#### Resolution (Pending)
+#### Resolution
 
-The following categories are areas that were initially candidates for `wildcard`, but after reviewing the benchmarking data, the fields will not typically have high enough cardinality to make them ideal candidates for `wildcard`.
+The following categories were initially candidates for `wildcard`, but after reviewing the benchmarking data, the fields will not typically have high enough cardinality to make them ideal candidates for `wildcard`.
 ##### File paths and names
 
-File path values are likely to compress extremely well as `keyword` since `keyword` fields have common-prefix-based compression (`wildcard` values are blocks of 32 values compressed into a single LZ4 blob). On top of the worse doc values compression, the number of `postings` also increases significantly due to n-grams.
+File path values are likely to compress well as `keyword` since `keyword` fields have common-prefix-based compression (`wildcard` values are blocks of 32 values compressed into a single LZ4 blob). On top of the worse doc values compression, the number of `postings` also increases significantly due to n-grams.
```

Comment:
```suggestion
File path values are likely to compress well as `keyword` since `keyword` fields have common-prefix-based compression (`wildcard` values are blocks of 32 values compressed into a single LZ4 blob). In addition to worsened doc values compression, the number of `postings` also increases significantly due to n-grams.
```